web/src/title/TitleMode.svelte

```diff
@@ -35,33 +42,51 @@
     }
   }
 
-  let projectList = getProjectList($appFocus);
+  let studyAreas = $projectStorage.studyAreaProjects();
 
-  function deleteProject(key: string) {
-    if (window.confirm(`Really delete project ${key}? You can't undo this.`)) {
-      window.localStorage.removeItem(key);
-      projectList = getProjectList($appFocus);
+  function loadProjectFromUrlParam(projectIDParam: string) {
+    let projectID = projectIDParam as ProjectID;
+    try {
+      let projectName = $projectStorage.projectName(projectID);
+      if (!projectName) {
+        console.error(`Project ${projectID} from URL not found`);
+        return;
+      }
+      loadProjectPrompt(projectID, projectName);
+    } catch {
+      console.error(`Error trying to fetch project from URL: ${projectID}`);
+    }
+  }
+
+  function deleteProject(projectID: ProjectID, projectName: string) {
+    if (
+      window.confirm(
+        `Really delete project ${projectName}? You can't undo this.`,
+      )
+    ) {
+      $projectStorage.removeProject(projectID);
+      studyAreas = $projectStorage.studyAreaProjects();
     }
   }
 
-  function renameProject(key: string) {
-    let newName = window.prompt(`Rename project ${key} to what?`, key);
+  function renameProject(projectID: ProjectID, existingName: string) {
+    let newName = window.prompt(
+      `Rename project ${existingName} to what?`,
+      existingName,
+    );
     if (newName) {
-      // TODO Again, hide leading ltn_?
-      if (!newName.startsWith("ltn_")) {
-        newName = `ltn_${newName}`;
+      try {
+        $projectStorage.renameProject(projectID, newName);
+      } catch (e) {
+        window.alert(`Couldn't rename project: ${e}`);
       }
-
-      let gj = window.localStorage.getItem(key)!;
-      window.localStorage.setItem(newName, gj);
-      window.localStorage.removeItem(key);
-      projectList = getProjectList($appFocus);
+      studyAreas = $projectStorage.studyAreaProjects();
     }
   }
 
-  async function loadProject(key: string) {
-    loading = `Loading project ${key}`;
-    await loadFromLocalStorage(key);
+  async function loadProjectPrompt(projectID: ProjectID, projectName: string) {
```

Comment:
nit: I wasn't sure what "prompt" meant. Is this meant to be "with a loading screen"?

Reply:
Yeah, it's not a great name. I updated in #247